docs/platforms/javascript/guides/nuxt/manual-setup.mdx

```diff
@@ -186,81 +229,107 @@ from trying to fetch missing files and avoiding unnecessary errors.
+  }
+</script>
+
+<template>
+  <button id="errorBtn" @click="triggerClientError">Throw Client Error</button>
+</template>
 ```
 
-If you want to test server-side monitoring locally, build your project and run:
-  ```
-  # Start your app after building your project with `nuxi build`
-  node --import ./.output/server/sentry.server.config.mjs .output/server/index.mjs
-  ```
+<OnboardingOption optionId="performance" hideForThisOption>
+  Open the page in a browser (for most Nuxt applications, this will be at
+  localhost) and click the button to trigger a frontend error.
+</OnboardingOption>
 
-<Alert level="warning">
-  Sentry server-side monitoring **doesn't work in development mode!**
+<PlatformContent includePath="getting-started-browser-sandbox-warning" />
 
-  If you experience any issues with the server-side setup, check out <PlatformLink to="/troubleshooting">Troubleshooting</PlatformLink>
-  or read through the different <PlatformLink to="/install">installation methods</PlatformLink>.
-</Alert>
+<OnboardingOption optionId="performance">
+### Tracing
 
-### Client-side
+To test tracing, create a test API route `server/api/sentry-example.get.ts`:
 
-Add the following snippet on the client-side:
+```js {tabTitle:Nitro} {filename:server/api/sentry-example.get.ts}
+export default defineEventHandler((event) => {
+  throw new Error("Sentry Example API Route Error");
+});
+```
+
+Then update the test page by including a new button that executes a function to fetch your API route:
 
 ```html {tabTitle:Vue} {filename:pages/example-error.vue}
 <script setup>
-  import * as Sentry from '@sentry/nuxt';
+  import * as Sentry from "@sentry/nuxt";
 
   function triggerClientError() {
     throw new Error("Nuxt Button Error");
-  };
+  }
 
   function getSentryData() {
     Sentry.startSpan(
       {
-        name: 'Example Frontend Span',
-        op: 'test'
+        name: "Example Frontend Span",
+        op: "test",
       },
       async () => {
-        await $fetch('/api/sentry-example');
+        await $fetch("/api/sentry-example");
       }
-    )
+    );
   }
 </script>
 
 <template>
-  <button id="errorBtn" @click="triggerClientError">
-    Throw Client Error
-  </button>
-  <button type="button" @click="getSentryData">
-    Throw Server Error
-  </button>
+  <button id="errorBtn" @click="triggerClientError">Throw Client Error</button>
+  <button type="button" @click="getSentryData">Throw Server Error</button>
 </template>
 ```
 
-<Alert>
+Once you have your test code in place, you need to build your project since Sentry's server-side monitoring doesn't work in development mode.
+Then start your app and make sure to load Sentry on the server side by explicitly adding it via `--import`.
 
-  Learn more about manually capturing an error or message in our <PlatformLink to="/usage/">Usage documentation</PlatformLink>.
+After running your project:
 
-</Alert>
+1. Open your test page in a browser (for most Nuxt applications, this will be at localhost)
```

Comment:
I am not a fan of just writing `localhost` as this is the same for every local application. But we could add the port here, as this is something that differers from setup to setup. E.g. Svelte uses [port 5173](https://svelte.dev/docs/kit/creating-a-project) as default and Nuxt [port 3000](https://nuxt.com/docs/api/nuxt-config#port).
```suggestion
1. Open your test page in a browser (for most Nuxt applications, this will be at localhost:3000)
```